src/main/java/com/coherent/training/selenium/stanila/employee/Employee.java

```diff
@@ -0,0 +1,23 @@
+package com.coherent.training.selenium.stanila.employee;
+
+public class Employee {
```

Comment:
what are you supposed to do with it without getters at least?

Reply:
Added getters.